deploy/operator.yaml

```diff
@@ -54,3 +54,6 @@ spec:
               value: "managed"
             - name: REBALANCE_PODS
               value: "true"
+            # this should be set for production and development via MT repo
+            - name: ALERT_SMTP_FROM
+              value: "default@test.com"
```

Comment:
If this is been set by the MT repo should the `ALERT_SMTP_FROM` be also added to the delorean configuration?

https://github.com/integr8ly/delorean/blob/master/configurations/managed-tenants-addons-config-rhoam.yaml

Reply:
Yes, I think so.
Good catch 👍